TestApp/app/screens/TransmissionScreen.js

```diff
@@ -127,6 +128,15 @@ export default class TransmissionScreen extends Component {
       />
     );
 
+    // After trying to fix the next line lint warning, the code was harder to read and format, disable it once.
+    // eslint-disable-next-line object-curly-newline
+    const settingsRenderItem = ({ item: { title, disabled, value, onChange } }) => (
```

Comment:
This would have been the only way to fix all errors without disabling the rule:

```
    const settingsRenderItem = ({ item }) => {
      const {
        title,
        disabled,
        value,
        onChange
      } = item;
      return (
        <View style={SharedStyles.item}>
          <Text style={SharedStyles.itemTitle}>{title}</Text>
          <Switch disabled={disabled} value={value} onValueChange={onChange} />
        </View>
      );
    };
```

This is not better according to me.